scanner/Makefile

```diff
@@ -36,10 +36,8 @@ EXPECTED_GO_VERSION := $(shell cat ../EXPECTED_GO_VERSION)
 
 BUILD_FLAGS := CGO_ENABLED=0 GOOS=${GOOS} GOARCH=${GOARCH}
```

Comment:
Actually, I think there are still issues with the build flags. When I build `scannerctl`, I get one built for `linux`, even though I have darwin. That's because we use `GOOS`, which is `linux`, instead of `HOST_OS`, which would be `darwin` for me

Reply:
Yes, [GOOS defaults to Linux](https://github.com/stackrox/stackrox/blob/jvdm%2Fe2e%2Fclient/make/env.mk#L34) If you call `make GOOS='$(HOST_OS)' <target>` you should be able to get  `go build` for your local OS.